app/components/AppNavBar.tsx

```diff
@@ -1,18 +1,20 @@
+import { User } from "@auth0/auth0-react";
 import Container from "react-bootstrap/Container";
-import Navbar from "react-bootstrap/Navbar";
-import Image from "react-bootstrap/Image";
 import Dropdown from "react-bootstrap/Dropdown";
-import { User } from "@auth0/auth0-react";
+import Image from "react-bootstrap/Image";
+import Col from "react-bootstrap/Col";
+import Navbar from "react-bootstrap/Navbar";
```

Comment:
these just got reorganized. til you can organize your imports from your code editor